frontend/app/src/lib/breadcrumbs.ts

```diff
@@ -102,7 +102,7 @@ const createRouteLabel = (path: TenantedPath): string => {
       return '';
     default:
       // eslint-disable-next-line no-case-declarations
-      const exhaustiveCheck: never = path;
+      const exhaustiveCheck: never = path as never;
```

Comment:
hehe, I think this change is actually flagging a real error - what's the TS error if we remove this type assertion?